webdriver-spec.html

```diff
@@ -5882,241 +5901,651 @@
+          <tr><td><code>\uE051</code></td><td><code>"Control"</code></td></tr>
+          <tr><td><code>\uE052</code></td><td><code>"Alt"</code></td></tr>
+          <tr><td><code>\uE053</code></td><td><code>"Meta"</code></td></tr>
+          <!-- Numpad Variants -->
+          <tr><td><code>\uE054</code></td><td><code>"PageUp"</code></td></tr>
+          <tr><td><code>\uE055</code></td><td><code>"PageDown"</code></td></tr>
+          <tr><td><code>\uE056</code></td><td><code>"End"</code></td></tr>
+          <tr><td><code>\uE057</code></td><td><code>"Home"</code></td></tr>
+          <tr><td><code>\uE058</code></td><td><code>"ArrowLeft"</code></td></tr>
+          <tr><td><code>\uE059</code></td><td><code>"ArrowUp"</code></td></tr>
+          <tr><td><code>\uE05A</code></td><td><code>"ArrowRight"</code></td></tr>
+          <tr><td><code>\uE05B</code></td><td><code>"ArrowDown"</code></td></tr>
+          <tr><td><code>\uE05C</code></td><td><code>"Insert"</code></td></tr>
+          <tr><td><code>\uE05D</code></td><td><code>"Delete"</code></td></tr>
+        </table>
 
-        <li><p>If <a>element</a> <a>is stale</a>,
-         return <a>error</a> with <a>error code</a> <a>stale element reference</a>.
+      <p>The <dfn>code</dfn> for <var>key</var> is the value in the
+      last column of the following table on the row with <var>key</var> in
+      either the first or second column, if any such row exists,
+      otherwise it is <code>undefined</code>.
 
-        <li><p>Run the <a>focusing steps</a> for the <a>element</a>.
+        <!-- TODO This should perhaps return undefined when the shift
+        key doesn't match the requirement for producing the character?
+        -->
 
-        <li><p>Let <var>active element</var> be the <a>active element</a> of
-          the <a>current browsing context</a>’s <a>document element</a>.
+        <table class=simple>
+          <tr>
+            <th>Key
+            <th>Alternate Key
+            <th>code
+          </tr>
+          <tr><td><code>"`"</code></td><td><code>"~"</code></td><td><code>"Backquote"</code><td></tr>
+          <tr><td><code>"\"</code></td><td><code>"|"</code></td><td><code>"Backslash"</code><td></tr>
+          <tr><td><code>"\uE003"</code></td><td><code></code></td><td><code>"Backspace"</code><td></tr>
+          <tr><td><code>"["</code></td><td><code>"{"</code></td><td><code>"BracketLeft"</code><td></tr>
+          <tr><td><code>"}"</code></td><td><code>"]"</code></td><td><code>"BracketRight"</code><td></tr>
+          <tr><td><code>","</code></td><td><code>"&lt;"</code></td><td><code>"Comma"</code><td></tr>
+          <tr><td><code>"0"</code></td><td><code>")"</code></td><td><code>"Digit0"</code><td></tr>
+          <tr><td><code>"1"</code></td><td><code>"!"</code></td><td><code>"Digit1"</code><td></tr>
+          <tr><td><code>"2"</code></td><td><code>"@"</code></td><td><code>"Digit2"</code><td></tr>
```

Comment:
@andreastt I previously read your very useful writeup here https://sny.no/2016/07/bttup. 
I made some investigation and I think it can be possible to import data from this project: https://github.com/Mottie/Keyboard/tree/master/layouts

Great part of work is done by many contributors and I'm sure if webdriver will start own keyboard layouts subproject people will help.